src/mame/machine/namco06.cpp

```diff
@@ -161,16 +161,21 @@ uint8_t namco_06xx_device::data_r(offs_t offset)
 
 
 void namco_06xx_device::data_w(offs_t offset, uint8_t data)
+{
+          machine().scheduler().synchronize(timer_expired_delegate(FUNC(namco_06xx_device::write_sync),this), data);
+}
```

Comment:
Indentation has gone whack on this line.

Reply:
Fixed.